src/main/java/org/spdx/maven/CreateSpdxMojo.java

```diff
@@ -478,9 +478,16 @@ public class CreateSpdxMojo extends AbstractMojo
      * If true, use ${project.groupId}:${artifactId} as the SPDX package name.
      * Otherwise, ${project.name} will be used
      */
-    @Parameter
+    @Parameter( property = "spdx.useArtifactID" )
```

Comment:
I've included the property here so the parameter can work running the goal via CLI.